test/support/test_helpers.ex

```diff
@@ -26,6 +26,11 @@ defmodule OAuth2.TestHelpers do
     end
   end
 
+  def unix_now do
```

Comment:
I copied this from `OAuth2.Util` because I think we shouldn't use the same functions from the code to test itself... but I can revert this if you wish.